DESCRIPTION

```diff
@@ -9,6 +9,7 @@ Authors@R: c(person("Ben", "Marwick", role = c("aut", "cre"), email = "benmarwic
              person("Martin", "Hinz", role = "ctb"),
              person("Clemens", "Schmid", role = "ctb"),
              person("Ricarda", "Braun", role = "ctb"))
+             person("Wojciech", "Francuzik", role = "ctb"))
```

Comment:
Missing comma in line 11 after `person("Ricarda", "Braun", role = "ctb"))`. Please add one.

Reply:
Oh - and a surplus bracket.